camera_frame:                          sr300_depth_optical_frame
velocity_scaling:                      0.1
distance_to_objects:                   -0.1
allowed_position_deviation:            0.02
allowed_rotation_deviation:            0.15

min_localization_x:                    0.2
min_localization_y:                    -0.15
min_localization_z:                    -0.05
max_localization_x:                    0.55
max_localization_y:                    0.13
max_localization_z:                    0.1

grasp_object:
  approaching_distance:                0.01
  current_threshold:                   70

place_object:
  force_threshold:                     3
  eef_step:                            0.0005
  incremental_movement_timeout:        15
  incremental_movement_distance:       0.1
  approaching_distance:                0.012
  approaching_distance_release:        0.05
